models/Limit.yaml

```
@@ -0,0 +1,12 @@
+title: EVSE limit
+allOf:
+  - type: object
+    properties:
+      type:
+        type: string
+        description: 'Limit type: time, energy, soc, range'
```

```suggestion
      type:
        type: string
        enum:
        - time
        - energy
        - soc
        - range
        description: 'Limit type: time, energy, soc, range'
```
Would also be nice to exten the description to cover what each of the different limits are, particularly the units

---

done